### DISCUSSION TITLE
Vacation and Sick Leave Tracker

### DISCUSSION THREAD

**Discussion by Abareyellow** [community/community#119696] [Programming Help]
### Body

Hello Everyone,

I am having some trouble starting a project that will track vacation time for employees. I am trying to figure how to store time in database. Also I want the database to update columns automatically. Is there a way SQL can do that or anything else can do that? Which languages/frameworks will help with this task? Any Resources to learn these things will be helpful.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by kalebers** [comment]
Hello,

- If you use SQL server you can use the SQL Server scheduler job or Transact-SQL.

Useful links:

- https://learn.microsoft.com/en-us/previous-versions/sql/sql-server-2008-r2/ms181153(v=sql.105)?redirectedfrom=MSDN
- https://learn.microsoft.com/en-us/sql/ssms/agent/schedule-a-job?view=sql-server-ver16&redirectedfrom=MSDN
